addons/apecloud-mysql/templates/_helpers.tpl

```diff
@@ -195,6 +195,7 @@ serviceRefDeclarations:
     serviceRefDeclarationSpecs:
       - serviceKind: etcd
         serviceVersion: "^*"
+    optional: true
```

Comment:
It seems etcd related info should not appear in apecloud-mysql.spec.common, because if proxy is not enabled, vttablet won't be launched.

Reply:
However, cmpd doesn't seem to be able to determine whether etcd is needed at the moment. When install addon, it doesn't know the actual state of the cluster. The current design is that if etcd is not needed, the variables rendered for etcd will be empty. To accommodate this, I set optional: true.